integration/bin/alluxio-yarn.sh

```diff
@@ -15,14 +15,15 @@
 #  alluxio-yarn.sh <numWorkers> <pathHdfs>
 
 function printUsage {
-  echo "Usage: alluxio-yarn.sh <numWorkers> <pathHdfs>"
+  echo "Usage: alluxio-yarn.sh <numWorkers> <pathHdfs> [masterAddress]"
```

Comment:
Changing the title to WIP until the docs are updated. I'll update this and the new client behavior after getting a first pass of review.